addons/analytic/static/src/components/analytic_distribution/analytic_distribution.js

```diff
@@ -563,7 +563,7 @@ export class AnalyticDistribution extends Component {
                 if (this.isDropdownOpen) {
```

Comment:
maybe a slightly better alternative is to check if the dropdown is empty here

```js
&& this.allPlans.length) {
```

or check if the focus is inside the dropdown (in the scenario the focus is on the cell containing the "No analytic plans found" dropdown)

```js
&& this.dropdownRef.el.contains(event.target))
```
because the current solution works, but in the future it could fail if by chance the element td/th outside the modal that we find, had a parent with an `id`